tests/unit/Statement/StatementListTest.php

```diff
@@ -275,7 +275,7 @@ public function testRemoveStatementsWithGuid_singleStatementRemoved() {
 		$this->assertEquals( $statements, $list->toArray() );
 	}
 
-	public function testRemoveStatementsWithGuid_multipleStatementsRemoved() {
+	public function testGivenGuidOfMulitpleStatements_multipleStatementsAreRemoved() {
```

Comment:
@thiemowmde even if you spot such typos in a case similar to this one, IMO you are wasting everyone's time by -1'ing and not just merging and then fixing it yourself